pkg/operator/ceph/cluster/crash/reconcile.go

```diff
@@ -117,6 +118,12 @@ func (r *ReconcileNode) reconcile(request reconcile.Request) (reconcile.Result,
 			logger.Errorf("more than one CephCluster found in the namespace %q, choosing the first one %q", namespace, cephCluster.GetName())
 		}
 
+		clusterImage := cephCluster.Spec.CephVersion.Image
+		cephVersion, err := version.GetImageVersion(clusterImage)
+		if err != nil {
+			logger.Warningf("could not identify ceph version for image %q used by cluster %q: %v", clusterImage, cephCluster.Name, err)
```

Comment:
Shouldn't we return here instead of passing an empty version?

Reply:
There was some agreement above that the lack of version tag should be a non-fatal error. I'm open to changing this if there's concern, though.

Reply:
Sorry I missed this https://github.com/rook/rook/pull/4366/files#diff-340a35fd504236d0f1e7840bac0c6460R90 in my review. All good.